modules/collect/src/main/java/com/opengamma/strata/collect/result/FailureItem.java

```diff
@@ -61,6 +63,11 @@
    */
   @PropertyDefinition(validate = "notEmpty")
   private final String message;
+  /**
+   * The attributes associated with this failure.
```

Comment:
A bit more detail would be nice here. Something like "Attributes can contain additional information about the failure. For example, a line number in a file or the ID of a trade".